### DISCUSSION TITLE
Return test results to external destination

### DISCUSSION THREAD

**Discussion by BapAtRb** [stepci/stepci#211] [Q&A] [answered]
Hey! 👋

I am wondering if it is possible to return the tests results to external destination like file or just simply pipe it from Step CI to further part of the script executing tests?
I have found #107, #108, #109 and #110 but (apart the HTML one) none of them have any updates. Are those functionalities planned to be implemented? I am just wondering what are plans for future if it comes to writing test output not only to screen but to other tools for further processing.
I have found #97 that gave me some hope but nothing is mentioned in the documentation, hence not sure if it was finally implemented.

Thanks in advance!

**Comment by mishushakov** [comment, marked as answer]
Hey, thanks for starting the discussion!

Unfortunately this is not supported currently and your best bet is to use the test runner directly.
